src/components/TimeSelect/TimeSelect.module.scss

```diff
@@ -8,7 +8,7 @@
 
     input {
       color: $color-1;
-      width: 52px !important;
+      width: 65px !important;
```

Comment:
@OskarJankowiak-Landingi ile tam trzeba w counterze, żeby się to nie rozjeżdżało?

Reply:
Poprawione na 58px. Jak zmieniam w dev toolsach to jest git i w edytorze i w SPA i w application-react.